iotedgedev/envvars.py

```diff
@@ -209,6 +209,26 @@ def save_envvar(self, key, value):
             self.output.error(f("Could not update the environment variable {key} in file {dotenv_path}"))
             sys.exit(-1)
 
+    def get_registries(self):
+        registries = {}
+        self.CONTAINER_REGISTRY = {}
+        for key in os.environ:
+            key = key.lower()
+            if key.startswith('container_registry_server'):
+                token = key[25:]
+                if token not in registries:
+                    registries[token] = {}
+                registries[token]['server'] = os.environ[key]
+            elif key.startswith(('container_registry_username', 'container_registry_password')):
+                token = key[27:]
+                subkey = key[19:27]
```

Comment:
There are a few magic numbers in this method (25, 27, 19, 27). Can you use variable or code (such as len(`'container_registry_server'`) to explain them?